src/lib/converter/types/intrinsic.ts

```diff
@@ -4,6 +4,20 @@ import {Type, IntrinsicType} from "../../models/index";
 import {Component, ConverterTypeComponent, ITypeTypeConverter} from "../components";
 import {Context} from "../context";
 
+// Copy typescript's @internal enum set from:
+// https://github.com/Microsoft/TypeScript/blob/v2.0.5/src/compiler/types.ts#L2297-L2298
```

Comment:
That makes sense. Unless it is needed in multiple places, defining it here is good with me.